radis/api/tools.py

```diff
@@ -317,3 +320,8 @@ def replace_PQR_with_m101(df):
             ),
             category=PerformanceWarning,
         )
+
+
+def root_dir():
```

Comment:
there is [getProjectRoot()](https://github.com/radis/radis/blob/6136932b371cc10ad32cdafe2a0a0ea0ff7c5e54/radis/misc/utils.py#L21) already